source/manual/alerts/signon-api-user-token-expires-soon.html.md

```diff
@@ -1,63 +1,86 @@
 ---
-owner_slack: "#govuk-2ndline-tech"
-title: Signon API User Token Expires Soon
+owner_slack: "#govuk-platform-engineering"
+title: Signon API user token expires soon
 parent: "/manual.html"
 layout: manual_layout
 section: Alertmanager alerts
 ---
 
-[signon]: https://signon.publishing.service.gov.uk/api_users
-[gds-api-adapters]: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api.rb
-
 One or more tokens for API Users are about to expire. You should rotate
 expiring tokens to ensure the associated application keeps working.
 
-- First login to [Signon], go to API Users and click on the appropriate API User.
+## Rotate a Signon API token
+
+> 🪨 This procedure is [toil](https://sre.google/workbook/eliminating-toil/) and should be eliminated or automated away. For now, it's unfortunately still a manual process.
+
+### Special cases
+
+If the token is for `Trade Tariff Admin` or `Trade Tariff Backend`, see [Trade Tariff Admin on the Wiki](https://gov-uk.atlassian.net/wiki/spaces/PLOPS/pages/3155099649/Trade+Tariff+Admin)
+
+If the token is for `Signon API Client (permission and suspension updater)`, talk with the [Access and Permissions team](https://gds.slack.com/channels/govuk-publishing-access-and-permissions-team).
+
+### 1. Issue a new token
+
+1. Go to the [APIs page](https://signon.publishing.service.gov.uk/api_users) in Signon.
+
+1. From the **API users** table, choose the API User whose **Email** matches the `api_user` field in the alert.
+
+    > Check the **Last synced at** time to see if the API User is still using the
+    > application. If you are sure the token is disused, you can choose
+    > **Revoke** to delete it and then you're done.
+
+1. Choose **Manage tokens**.
+
+1. Choose **Add application token**.
+
+1. From the **Application** dropdown, select the application that matches the `application` field in the alert.
+
+1. Choose **Create access token**.
+
+The nightly `signon-sync-token-secrets-to-k8s` cronjob will update the token in the Kubernetes secret that the application uses.
+
+### 2. Check that the client application works with the new token
+
+1. Trigger the token sync cronjob to run now (or wait until the next day).
 
-> Check the *Last synced at* time to see if the API User is still using the
-> application. If you are confident the token is unused, the you can just
-> click the *Revoke* button to remove it and there's no need to continue.
+    ```sh
+    k create job --from cronjob/signon-sync-token-secrets-to-k8s signon-token-sync-$USER
```

Comment:
I went through the commands and they worked after adding `apps` namespace to the command, could this be added to the command or add a command to set the namespace in the docs.

Reply:
Thanks for testing the steps!

We get users to set their context namespace to `apps` in the [getting started guide](https://docs.publishing.service.gov.uk/kubernetes/get-started/access-eks-cluster/#access-a-cluster-for-the-first-time) and the [quick-reference guide](https://docs.publishing.service.gov.uk/kubernetes/cheatsheet.html#prerequisites) so that we don't have to repeat `-n apps` in hundreds of examples. Definitely let me know if there's some other place where it's missing from setup instructions tho!

Reply:
ok, thats fine then, I guess I'm a bit different as I mainly use the `datagovuk` space for things

Reply:
Ah yeah that's a fair point, it's a little inconvenient if your day-to-day is mostly in a different namespace, sorry :( You could have another context or use the [ns](https://github.com/ahmetb/kubectx) kubectl plugin (I recommend installing via [krew](https://krew.sigs.k8s.io/docs/user-guide/quickstart/)) to help with this a bit.